tools/stress-cluster/cluster/README.md

```diff
@@ -112,6 +113,12 @@ in an Azure AKS cluster (shared or personal).
 kind create cluster
 ```
 
+# Stress test addons
+Steps for deploying the stress test addons helm chart:
+1. Increment the version number in Chart.yaml (e.g. 0.1.0 -> 0.1.1)
+1. Run ./deploy.ps1
+1. Update all the helm chart versions for stress-test-addons refernces in stress-cluster/chaos/examples/
+1. Run azure-sdk-tools\eng\common\scripts\stress-testing\deploy-stress-tests.ps1 script in the examples directory, this will update all the nested helm charts (-login tag is needed for the first run)
```

Comment:
Github link to the examples directory here. Also, add that one should validate the tests succeeded (e.g. wait for pod to reach Running/Completed state vs. Error state).

Reply:
And then, another list item here to specify to update all the stress tests across the other repos in the same manner.